SPECS/cal10n/cal10n.spec

```diff
@@ -16,16 +15,15 @@
 #
 Summary:        Compiler assisted localization library (CAL10N)
 Name:           cal10n
-Version:        0.7.7
-Release:        6%{?dist}
+Version:        0.8.1
+Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          Development/Libraries/Java
 URL:            http://cal10n.qos.ch
-Source0:        https://github.com/qos-ch/cal10n/archive/refs/tags/v_%{version}.tar.gz#/%{name}-%{version}.tar.gz
-Source1:        build.xml-0.7.7.tar.xz
-Patch0:         cal10n-0.7.7-sourcetarget.patch
+Source0:        %{_distro_sources_url}/cal10n-0.8.1.tar.xz
```

Comment:
These should point to the upstream source URL, not our mirror of the sources (`%{_distro_sources_url}`) / `https://cblmarinerstorage.blob.core.windows.net/sources/core`

Reply:
I updated this spec based on openSUSE's v0.8.1 spec, but their source tarball has an extra folder compared to the upstream source (https://github.com/qos-ch/cal10n/archive/v_0.8.1.tar.gz). 

Do you happen to know how openSUSE generates their source tarballs?